src/CRM/CivixBundle/Resources/views/Code/mkdocs.yml.php

```diff
@@ -0,0 +1,21 @@
+site_name: <?php echo "$fullName\n"; ?>
```

Comment:
Maybe leave a trail pointing to some information about what how to use mkdocs? e.g.

```suggestion
## Use mkdocs to generate a manual for this extension. For more information about mkdocs,
## see https://docs.civicrm.org/dev/en/latest/documentation/

site_name: <?php echo "$fullName\n"; ?>
```

Reply:
Good call. I've updated it in PR #162, which just has the first commit introducing these files with the old mkdocs.yml extensions syntax. I'll rebase this one once that one gets merged.